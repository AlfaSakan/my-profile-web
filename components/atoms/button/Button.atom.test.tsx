import renderer from 'react-test-renderer';
import Button from './Button.atom';

describe('buttom component test', () => {
  test('should render correctly', () => {
    const tree = renderer
      .create(<Button onClick={() => {}} text="Text" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('should render correctly', () => {
    const tree = renderer.create(<Button text="Text" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
