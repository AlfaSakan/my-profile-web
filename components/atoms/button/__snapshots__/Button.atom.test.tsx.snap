// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`buttom component test should render correctly 1`] = `
<button
  className="flex items-center justify-center w-32 h-12 rounded-lg bg-secondary"
  onClick={[Function]}
  style={
    Object {
      "height": undefined,
      "width": undefined,
    }
  }
>
  <p>
    Text
  </p>
</button>
`;

exports[`buttom component test should render correctly 2`] = `
<button
  className="flex items-center justify-center w-32 h-12 rounded-lg bg-secondary"
  style={
    Object {
      "height": undefined,
      "width": undefined,
    }
  }
>
  <p>
    Text
  </p>
</button>
`;
